script/data-harmonizer/export_utils.js

```diff
@@ -165,19 +165,22 @@ Object.assign(DataHarmonizer, {
 	getTransformedField: function (headerName, value, field, prefix) {
 
 		if (field.sources) {
-			term = DataHarmonizer.schema.enums[field.sources].permissible_values[value];		
+			// iterate thru and will return the first match found in field.sources
+			for ( source in field.sources) {
+				term = DataHarmonizer.schema.enums[field.sources].permissible_values[value];
```

Comment:
I think field.sources here should be "source"?

Reply:
Good catch. My bad